File: Introduccion a Poo/Ejercicio 2/Python/Empleado.py
class Empleado:
    def __init__(self, nombre, sueldo):
        self.nombre = nombre
        self.sueldo = sueldo
    def sueldo_anual(self):
        return self.sueldo * 12
    def aumento(self):
        self.sueldo = self.sueldo * 1.10
e1 = Empleado("Nicole", 1000)
e2 = Empleado("Marcelo", 1200)
print(e1.sueldo)
print(e2.sueldo)
e1.aumento()
e2.aumento()
print(e1.sueldo)
print(e2.sueldo)

File: Introduccion a Poo/Ejercicio 2/Java/Empleado.java
public class Empleado {
    private String nombre;
    private double sueldo;
    public Empleado(String nombre, double sueldo) {
        this.nombre = nombre;
        this.sueldo = sueldo;
    }
    public double sueldoAnual() {
        return sueldo * 12;
    }
    public void aplicarAumento() {
        sueldo = sueldo * 1.10;
    }
